src/content/docs/en/reference/experimental-flags/sessions.mdx

```diff
@@ -22,83 +22,151 @@ This experimental feature allows you to store and access items such as login sta
+Some drivers may need extra packages to be installed. For example, Upstash requires the `@upstash/redis` package. Some drivers may also require environment variables or credentials to be set.
+
+<ReadMore>
+  See [the unstorage driver docs](https://unstorage.unjs.io/drivers) for more information.
+</ReadMore>
 
-<ReadMore>See [the unstorage driver docs](https://unstorage.unjs.io/drivers) for more information.</ReadMore>
 :::
 
+Set the driver name in the `session.driver` property. You can find the driver name in the unstorage documentation for the driver you want to use.
+
+```js title="astro.config.mjs" ins={4}
+  {
+    adapter: vercel(),
+    session: {
+      driver: "upstash",
+    },
+    experimental: {
+      session: true,
+    },
+  }
+```
+
 ### Driver options
 
-You can also pass any available options to the unstorage driver in a separate `session.options` object. The following example configures the [Netlify Blobs](https://unstorage.unjs.io/drivers/netlify) driver while also providing a `name` and specifying a `consistency` mode:
+You can also pass any available options to the unstorage driver in a separate `session.options` object. See the driver documentation for available options. For example, to set the base key for sessions in Upstash:
 
-```js title="astro.config.mjs" ins={7-11}
+```js title="astro.config.mjs" ins={5-7}
   {
-    adapter: netlify(),
-    experimental: {
-      session: {
-        // The name of the unstorage driver is kebab-case
-        driver: "netlify-blobs",
-        options: {
-          name: 'astro-sessions',
-          // Sessions need strong consistency
-          consistency: 'strong',
-        }
+    adapter: vercel(),
+    session: {
+      driver: "upstash",
+      options: {
+        base: "sessions",
       },
     },
+    experimental: {
+      session: true,
+    },
+  }
+```
+
+### Other session options
+
+You can configure additional options for sessions in the `session` object. 
+
+#### `session.cookie`
+
+<p>
+
+**Type:** `string` | `object`<br />
+**Default:** `astro-session`<br />
+
+</p>
+
+Configures the session cookie. This cookie is set in the response when a session is generated. No actual user data is stored in the cookie – just an ID that is used to identify a user's session. The `session.cookie` option can be used to set options for this cookie. You can either provide a `string`, which will be used as the cookie name, or an object which allows additional options: 
+
+```js title="astro.config.mjs" {4} ins={6-9}
+  {
+    session: {
+      // If set to a string, this will be used as the cookie name
+      // cookie: "my-session-id",
+      // If set to an object, this will allow advanced options to be set
+      cookie: {
+        name: "my-session-id"
+        sameSite: "Strict",
+      },
+    }
+  }
+```
+
+#### `session.ttl`
+
+<p>
+  
+**Type:** `number`<br />
+**Default:** `undefined`<br />
+
+</p>
+
+An optional default time-to-live for session values, in seconds. 
```

Comment:
```suggestion
An optional default time-to-live expiration period for session values, in seconds.
```
Adjust the wording if that's not exactly right, but the definition of a value should ideally be more than a restatement of its name. (ttl = ttl.) So, throwing the notion of an "expiration period" up there instead!